tests/b2b-mp/storefront/tests/product-search/B2B-MP-S2-product-search-by-sku.js

```diff
@@ -0,0 +1,22 @@
+import { SharedProductSearchBySkuScenario } from "../../../../cross-product/storefront/scenarios/product-search/shared-product-search-by-sku-scenario.js";
+import { loadDefaultOptions } from "../../../../../lib/utils.js";
+
+export const options = loadDefaultOptions();
+
+options.scenarios = {
+    S2_Product_Search: {
+        exec: 'executeProductSearchPageScenario',
+        executor: 'shared-iterations',
+        tags: {
+            testId: 'S2',
+            testGroup: 'ProductSearch',
+        },
+        iterations: 10
+    },
+};
+
+const productSearchPageScenario = new SharedProductSearchBySkuScenario('B2B_MP');
+
+export function executeProductSearchPageScenario() {
+    productSearchPageScenario.execute();
+}

```

Comment:
I wonder why we have these files in this PR if they have been merged already in https://github.com/spryker/commerce-k6-performance-tests/pull/6? Maybe master up-merge will help?

Reply:
I did it as part of my branch because I wanted to write a test to see how it works for me. It took me ~30min. I then simply gave the code to Yuliia via git cherry-pick to save her some work.